offchain-modules/src/packages/core.ts

```diff
@@ -1,7 +1,7 @@
 import { Config } from './config';
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const CKB = require('@nervosnetwork/ckb-sdk-core').default;
-import { CkbIndexer } from '@force-bridge/ckb/tx-helper/indexer';
```

Comment:
add `import 'module-alias/register';` in the beginning of `cli/index.ts` should fix this problem.